### PR TITLE
better matcher

### PR DESCRIPTION
**Pull request by binamov**

### CODE REVIEW

**Comment by binamov** [comment]
The usecase we're solving for:
- as a security operator, we'll provide a whitelist of regions for KMS
- the deployment under test should have its KMS in one, few, or all of those whitelisted regions

the `cmp` matcher was checking for all regions to be present
the `be_in` matcher helps check for one, or a few, not necessarily all 

@bgeesaman thoughts?

**Comment by aaronlippold** [comment]
the be_in compares two sets have the same elements regardless of order

**Comment by binamov** [comment]
@aaronlippold are you saying we can clean up some of those `.sort`s?

**Comment by aaronlippold** [comment]
Yes I am :)
--------
Aaron Lippold

lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

File: controls/3.5.rb [hunk 1]
```diff
@@ -164,9 +164,9 @@
   end
   keyring_locations.uniq!.sort!
 
-  describe "[#{gcp_project_id}] KMS Regions" do
+  describe "[#{gcp_project_id}] KMS Regions #{ keyring_locations.sort }" do
     subject { keyring_locations }
-    it { should cmp kms_regions_list.sort }
+    it { should be_in kms_regions_list.sort }
```
Comment (aaronlippold) [inline controls/3.5.rb:169]: No need to sort here unless you just want it more readable but yes I think this will work ...

Comment (binamov) [reply controls/3.5.rb:169]: keeping sorts for humans

**Comment by aaronlippold** [comment]
Hi,

Can I get a script that would setup my GCP env to allow all the controls to
run without error. I am still getting 38 profile errors on the controls in
the 'mitre-saf' gcp env.

I would like to have a clean env to run in to help push up some suggestions.

Thanks,
--------
Aaron Lippold

lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

**Pull request merged**